### DISCUSSION TITLE
DensifyPointCloud failure in WebODM pipeline

### DISCUSSION THREAD

**Discussion by joe-stroup** [cdcseacave/openMVS#1113] [Q&A]
Attached is the console.txt output from a WebODM task where the DensifyPointCloud step of the pipeline has failed.

This task was run on an Ubuntu/Docker version of WebODM, on a i7 11700 CPU with 64gb of RAM and a RTX3050 gpu.

This same task works OK on a Raspberry Pi 5 - which suggests a hardware issue.

I've found that some process in the WebODM pipeline fail with the turboboost feature if the i7 enabled.  I now keep this feature disabled in the machines bios.  I'm now wondering if the issue may have something to do with the RTX3050 gpu - an obvious difference with the Raspberry Pi 5.

Please let me know if I can provide additional information.

Thank you - Joe Stroup
joe.stroup@gmail.com
www.coptercam.tech

[console (5).txt](https://github.com/cdcseacave/openMVS/files/14450484/console.5.txt)

**Comment by cdcseacave** [comment]
Is the densified point-cloud saved to disk and has 112728839 points?
